ambry-rest/src/main/java/com.github.ambry.rest/NettyMetrics.java

```diff
@@ -237,5 +247,24 @@ public NettyMetrics(MetricRegistry metricRegistry) {
     // NettyServer
     nettyServerShutdownTimeInMs = metricRegistry.histogram(MetricRegistry.name(NettyServer.class, "ShutdownTimeInMs"));
     nettyServerStartTimeInMs = metricRegistry.histogram(MetricRegistry.name(NettyServer.class, "StartTimeInMs"));
+    // ConnectionStatsHandler
+    connectionsConnectedCount =
+        metricRegistry.counter(MetricRegistry.name(NettyServer.class, "ConnectionsConnectedCount"));
+    connectionsDisconnectedCount =
+        metricRegistry.counter(MetricRegistry.name(NettyServer.class, "ConnectionsDisconnectedCount"));
+  }
+
+  /**
+   * Register {@link ConnectionStatsHandler#openConnections} to track open connections
+   * @param openConnectionsCount open connections count to be tracked
+   */
+  public void registerConnectionsStatsHandler(AtomicLong openConnectionsCount) {
+    openConnections = new Gauge<Long>() {
+      @Override
+      public Long getValue() {
+        return openConnectionsCount.get();
+      }
+    };
+    metricRegistry.register(MetricRegistry.name(NettyServer.class, "OpenConnections"), openConnections);
```

Comment:
The owner should be the  one who actually owns the metric which is `ConnectionStatsHandler`.